source/features/go-to-action-from-file.tsx

```diff
@@ -15,7 +15,7 @@ function init(): void {
 		.replace(/["']/g, '')
 		.trim();
 
-	const actionURL = new URL(`${location.origin}/${getRepoURL()}/actions`);
+	const actionURL = new URL(`/${getRepoURL(true)}/actions`);
```

Comment:
Actually this could be an opportunity make it even more useful:
```suggestion
	const actionURL = new URL(getRepoURL('actions'));
```

- `getRepoURL()` without argument would be used for API calls, selectors and cache keys
- `getRepoURL(path)` would be use to generate URLs to user pages.

It could go one step further and accept an extra argument, these would be equivalent
```js
	const actionURL = new URL(getRepoURL('actions'));
	const actionURL = getRepoURL('actions', URL);
```

Reply:
Done. URL creation is now less noisy too.

```diff
- const url = `/${getRepoURL()}/actions`
+ const url = getRepoURL('actions')
```